DESIGN.md

```diff
@@ -0,0 +1,60 @@
+# Bulk Data Extract (BDE) Processor design
+
+## User requirements
+
+-  The _user interface_ is a read-only PostgreSQL database in AWS RDS, with
+   Azure authentication
+-  _Accessible_ to all
+   [current BDE users](https://github.com/linz/ds-infra-team/wiki/BDE-users)
+-  Process BDE files
+   _[every day](https://toitutewhenua.atlassian.net/wiki/spaces/GEOD/pages/86418416/BDE+Processor+Product)
+   except Sunday_
+-  Optional _post-processing_ pulled in from separate internal repository
+-  All changes for a single day are imported within a _single transaction_
+-  _Immediate notification_ of completed import (like the legacy BDE Processor
+   SNS topic)
```

Comment:
Also need to transform BDE data to a set of 'simplified' data. Mostly de-normalization, but also some other transform operations.

Reply:
That's part of the "Process BDE files" above - I've clarified it, but I want to declare the internal processing steps separately from the overall requirements.